src/Pages/Users/MyReservations.jsx

```diff
@@ -26,7 +26,7 @@ const MyReservations = () => {
 
   useEffect(() => {
     GetReservationStats(setError, setReservations);
-  }, [reservation, reservations]);
+  }, [reservation, setReservations]);
```

Comment:
I'm confused about how you listen to the method change which is never going to be changed. `setReservations` has to mutate the `reservations`, and it's not going to be changed itself, am I right?

Reply:
Hi @aimalamiri you are right, setReservations is a setter function that is guaranteed to be stable by useState, I must have added it by mistake to remove eslint error before passing it to API request function. I will remove it now.